platform/install/environments/gcp.mdx

```diff
@@ -215,72 +223,78 @@ EOF
 </Step>
 
 <Step>
-Once the service is active, obtain the external IP address:
+After the service is active, obtain the external IP address:
 
 ```bash
 kubectl get svc vcluster-platform-loadbalancer -n vcluster-platform
 ```
 
-and navigate to the IP address in your browser `https://<EXTERNAL_IP>`.
+Navigate to the IP address in your browser `https://<EXTERNAL_IP>`.
 
-:::tip certificate
-Note that the platform uses a self-signed certificate, so you need to
-accept the warning in your browser. For production use, you should [replace the certificate with a valid one.](/platform/configure/domain#configure-tls)
+:::tip
+The platform uses a self-signed certificate, so you must
```

Comment:
It should be something that they will see the certificate warning in the browser and will have to click accept to see the page. _must_ sounds a bit bad in this context (it's security related).